src/main/webapp/ts/matches.ts

```diff
@@ -2,12 +2,18 @@ function loadMatches() {
     fetch('/matches')
         .then(response => response.json())
         .then((matches) => {
+            if(matches != null) {
             const matchListElement = document.getElementById('match-history')!;
             matchListElement.innerHTML = "";
 
             matches.forEach((match: User) => {
                 matchListElement.appendChild(createMatchElement(match));
             })
+            } else {
+                //redirect to logged out homepage because user is not logged in
```

Comment:
maybe fetch the login url from the /userapi servlet and redirect to that instead?

Reply:
Good idea, I changed this.